crates/dojo-lang/src/plugin_test_data/system

```diff
@@ -84,6 +84,62 @@ mod withcomponent {
         testcomponent2_event: testcomponent2::Event,
     }
 }
+#[dojo::interface]
+trait IEmptyTrait;
+
+#[dojo::interface]
+trait IActions {
+    /// Even if it looks weird, the expected pattern for a constant in a trait is
+    /// `const <name>: <type>;`
+    /// See: https://github.com/starkware-libs/cairo/blob/600553d34a1af93ff7fb038ac677efdd3fd07010/crates/cairo-lang-parser/src/parser.rs#L835
+    const ONE: u8;
```

Comment:
We may want to write an other test where everything works as expected, to ensure the normal behavior is well supported.

Reply:
Ok, I splitted this trait into 2 traits: one "faulty" with a constant inside, and one with correct functions only.